Benchmarks test failing in GitHub Actions.

Benchmarks test fail with 
`
Failed: trying different commit/environment
Failed to build the project and import the benchmark suite.
Error: Process completed with exit code 1.
`


```
Run asv run --config asv.conf.actions.json
  asv run --config asv.conf.actions.json
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.1[2](https://github.com/sympy/sympy/actions/runs/7935642409/job/21669865896?pr=26238#step:10:2).2/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/[3](https://github.com/sympy/sympy/actions/runs/7935642409/job/21669865896?pr=26238#step:10:3).12.2/x6[4](https://github.com/sympy/sympy/actions/runs/7935642409/job/21669865896?pr=26238#step:10:4)/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.2/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.2/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.12.2/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.12.2/x64/lib
Couldn't load asv.plugins._mamba_helpers because
No module named 'libmambapy'
· Creating environments
· Discovering benchmarks
·· Uninstalling from virtualenv-py3.8-pip+fastcache-pip+mpmath-pip+numpy
·· Building a00718ba for virtualenv-py3.8-pip+fastcache-pip+mpmath-pip+numpy
·· Installing a00718ba into virtualenv-py3.8-pip+fastcache-pip+mpmath-pip+numpy
·· Failed: trying different commit/environment
·· Uninstalling from virtualenv-py3.8-pip+fastcache-pip+mpmath-pip+numpy
·· Building 101e6d02 for virtualenv-py3.8-pip+fastcache-pip+mpmath-pip+numpy
·· Installing 101e6d02 into virtualenv-py3.8-pip+fastcache-pip+mpmath-pip+numpy
·· Failed: trying different commit/environment
·· Uninstalling from virtualenv-py3.8-pip+fastcache-pip+mpmath-pip+numpy
·· Building [5](https://github.com/sympy/sympy/actions/runs/7935642409/job/21669865896?pr=26238#step:10:5)f11bbf4 for virtualenv-py3.[8](https://github.com/sympy/sympy/actions/runs/7935642409/job/21669865896?pr=26238#step:10:8)-pip+fastcache-pip+mpmath-pip+numpy
·· Installing 5f[11](https://github.com/sympy/sympy/actions/runs/7935642409/job/21669865896?pr=26238#step:10:12)bbf4 into virtualenv-py3.8-pip+fastcache-pip+mpmath-pip+numpy
·· Failed to build the project and import the benchmark suite.
Error: Process completed with exit code 1.
```

--- a/asv.conf.actions.json
+++ b/asv.conf.actions.json
@@ -17,6 +17,13 @@
     // project being benchmarked
     "repo": ".",
 
+    // The command to run to build the project.
+    // https://github.com/sympy/sympy/issues/26239
+    "build_command": [
+        "python -m pip install build",
+        "python -m build --wheel -o {build_cache_dir} {build_dir}"
+    ],
+
     // List of branches to benchmark. If not provided, defaults to "master"
     // (for git) or "tip" (for mercurial).
 

